changelog.txt

```diff
@@ -1,3 +1,49 @@
+Feature: Improved logic of sharing industry production between 3 or more stations (#7922)
+Feature: Highlight the item under mouse cursor in file browser (#7900)
+Feature: [GS] Methods to change town rating of companies (#7898)
+Feature: [NewGRF] Callback profiling command (#7868)
+Feature: Add a setting to show the name of the NewGRF of a vehicle in the build window (#7852)
+Feature: Ability to filter industry directory window by cargo (#7843)
+Feature: Minimap screenshot type (#7817)
+Feature: [GS] Methods to control engine availability of a specific company (#7791)
+Feature: Configurable game ending year (#7747)
+Feature: Separate window for taking screenshots (#7550)
+Change: Move autorenew setting to basic category (#7984)
+Change: Improved algorithm for transfer feeder payments (#7935)
+Change: Show volume sliders with wedges instead of boxy slider (#7902)
+Change: Auto-restart loads the original resources (e.g. save or scenario) again (#7688)
+Change: Improve readability of integer lists saved to config files (#7396)
+Fix #7976: Crash when attempting to kick the host via rcon (#7985)
+Fix #7592: Road vehicles no longer balanced between multiple road stop stations (#7979)
+Fix: Station rating effects affecting too large area (#7977)
+Fix #7974: Crash when Ctrl+click to show a collapsed vehicle group (#7975)
+Fix #7969: Crash when executing a recurive console alias (#7973)
+Fix #6566: Very long loading of the maximum "zoom out" level at high resolutions (#7968)
+Fix #7952: Crash when switching input languages (#7953)
+Fix #7927: Incorrect resolution shown in "Huge screenshot" query window (#7949)
+Fix: [OSX] Don't show a crash/assertion message box for a GUI-less video driver (#7934)
+Fix #7925: Corrupt savegames could lead to corruption of the titlegame (#7932)
+Fix: [Fluidsynth] Music notes from previous song were not properly reset (#7930)
+Fix: Invalid string usage within music window (#7928)
+Fix: Non-deterministic name sorting in industry directory window (#7915)
+Fix #7899: Various issues with town list window sorting (#7906, #7916)
+Fix #7587: Fix possible crashes when loading old save games with invalid waypoint positions (#7894)
+Fix: Avoid a crash by properly resetting timetable duration when loading old savegames (#7894)
+Fix: Possible crash when post road-works cleanup removes all road pieces (#7903)
+Fix #7891: Fix crash when loading save from 1.7.2 (#7892)
+Fix #7887: Missing sound effects for some main toolbar buttons (#7888)
+Fix #6667: Avoid confusion by also recalculating bridge costs for 'spectated' AI companies (#7884)
+Fix: Allow old NewGRF industries to hide in/out cargo slots (#7882)
+Fix: [Windows] Fix bootstrap GUI with Uniscribe but no Freetype (#7878)
+Fix: Missing keycodes for hotkeys.cfg (#7850)
+Fix #7625: Ensure road infrastructure cost is correctly updated when upgrading your own roads (#7628)
+Fix #7088: Close AI/GS textfile windows when their data becomes invalid (#7094)
```

Comment:
Could be "Close AI/GS textfile windows when their data **become** invalid"

Reply:
nope, "data" is plural

Reply:
So you are technically correct actually, but I don't like how it reads. So I just rewrote the message to avoid "data" entirely :)